### PR TITLE
Problem: chain-main protobuf and gRPC definitions are missing (fixes #36)

### LINKED ISSUE #36: Problem: chain-main protobuf and gRPC definitions are missing
e.g. the x/nft module

### PR DESCRIPTION
**Pull request by tomtau**
Solution:
- removed protobuf generated files and switched to DeFi Wallet Core which supports chain-main messages and queries

### CODE REVIEW

**Review by leejw51crypto — APPROVED** [review]
lgtm

File: build.py [hunk 1]
```diff
@@ -0,0 +1,58 @@
+import os
+import shutil
+from distutils.core import Distribution
+from pathlib import Path
+from subprocess import call
+
+from setuptools_rust import Binding, RustExtension, build_rust
+
+
+class BuildFailed(Exception):
```
Comment (macong-cdc) [inline build.py]: This class seems to be unused.

**Review by macong-cdc — APPROVED** [review]
LGTM.

File: build.py (hunk 1 above)
Comment (tomtau) [reply build.py]: it was for testing: https://github.com/crypto-org-chain/chainlibpy/pull/52/files#diff-b70bfb027a8ef3ec4ab07a53a3b8ba83e63334ba3ce5ca6d9d773900ed452e5eR58 -- I can perhaps comment it out too

Comment (macong-cdc) [reply build.py]: I see, thank you.

**Pull request merged**